### PR TITLE
update doc for model compression

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: docs/en_US/Compressor/Overview.md [hunk 1]
```diff
@@ -1,182 +1,172 @@
 # Compressor
-NNI provides easy-to-use toolkit to help user  design and use compression algorithm.
+NNI provides an easy-to-use toolkit to help user design and use compression algorithms. It supports Tensorflow and Pytorch with unified interface. For users to compress their models, they only need to add several lines in their code. There are some popular model compression algorithms built-in in NNI. Users could further use NNI's auto tuning power to find the best compressed model, which is detailed in [Auto Model Compression](./AutoCompression). On the other hand, users could easily customize their new compression algorithms using NNI's interface, refer to the tutorial [here](#CustomizeCompression).
```
Comment (liuzhe-lz) [inline docs/en_US/Compressor/Overview.md]: PyTorch

File: docs/en_US/Compressor/Overview.md [hunk 2]
```diff
@@ -1,182 +1,172 @@
...
 
-Following our default format, user can set configure in his code or a yaml file. And pass configure to compressor by init() or load_configure()
+* __op_types__: This is to specify what types of operations to be compressed. 'default' means following the algorithm's default setting.
+* __op_names__: This is to specify by name what operations to be compressed. If this field is omitted, operations will not be filtered by it.
+* __exclude__: Default is False. If this field is True, it means the operations with specified types and names will be excluded from the compression.
 
+There are also other keys in the `dict`, but they are specific for every compression algorithm. For example, some , some.
 
-Code 
+The configuration in each `dict` is applied one by one, that is, latter configuration will overwrite former ones on the operations that are within the scope of both of them.
```
Comment (liuzhe-lz) [inline docs/en_US/Compressor/Overview.md]: in each dict?

File: docs/en_US/Compressor/Overview.md [hunk 3]
```diff
@@ -1,182 +1,172 @@
...
-        # design your mask and return your mask
-        return your_mask
-    
-    # you can also design your method help to generate mask
-    def your_method(self, your_input):
-        #your code
 ```
 
-if you want to generate new weight and replace the old one, you are suggested to use Quantizer as base class. And you can manage the weight in quantize_weight() method.
+For the simpliest algorithm, you can only override `calc_mask`, it receives each layer's information (i.e., layer name) and the layer's weight, you generate the mask for this weight in this function and return. Then NNI applies the mask for you.
```
Comment (liuzhe-lz) [inline docs/en_US/Compressor/Overview.md]: "can only"

File: docs/en_US/Compressor/Overview.md [hunk 4]
```diff
@@ -1,182 +1,172 @@
...
+
+Some algorithms generate mask based on training progress, i.e., epoch number. We provide `update_epoch` for the pruner to be aware of the training progress.
+
+Some algorithms may want global information for generating masks, for example, all weights of the model (for statistic information), model optimizer's information. NNI supports this requirement using `bind_model`. `bind_model` receives the complete model, thus, it could record any information (e.g., referennce to weights) it cares about. Then `step` can process or update the information according to the algorithm. You can refer to [source code of built-in algorithms](https://github.com/microsoft/nni/tree/master/src/sdk/pynni/nni/compressors) for example implementations.
```
Comment (liuzhe-lz) [inline docs/en_US/Compressor/Overview.md]: referennce -> reference

File: docs/en_US/Compressor/Quantizer.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 Quantizer on NNI Compressor
 ===
+
+<a name="NaiveQuantizer"></a>
```
Comment (liuzhe-lz) [inline docs/en_US/Compressor/Quantizer.md]: Shouldn't github add anchors automatically?

File: docs/en_US/Compressor/Overview.md (hunk 2 above)
Comment (QuanluZhang) [reply docs/en_US/Compressor/Overview.md]: what term do you think to use here?

File: docs/en_US/Compressor/Overview.md (hunk 3 above)
Comment (QuanluZhang) [reply docs/en_US/Compressor/Overview.md]: thx, changed to "you only need to"

File: docs/en_US/Compressor/Overview.md (hunk 4 above)
Comment (QuanluZhang) [reply docs/en_US/Compressor/Overview.md]: fixed

File: docs/en_US/Compressor/Quantizer.md (hunk 5 above)
Comment (QuanluZhang) [reply docs/en_US/Compressor/Quantizer.md]: how?

File: docs/en_US/Compressor/Overview.md (hunk 2 above)
Comment (QuanluZhang) [reply docs/en_US/Compressor/Overview.md]: fixed, thx.
